pages/common/distccd.md

```diff
@@ -0,0 +1,24 @@
+# distccd
+
+> Server daemon for the distcc distributed compiler.
+> More information: <https://distcc.github.io>.
+
+- Start a daemon with default settings:
+
+`distccd --daemon`
+
+- Start a daemon, accepting connections from private network IPv4 ranges:
+
+`distccd --daemon --allow-private`
+
+- Start a daemon, accepting connections from a specific network address or address range:
+
+`distccd --daemon --allow {{ip_address|network_prefix}}`
+
+- Start a daemon with a maximum of 4 tasks at any time and lowered priority (niceness = 5):
```

Comment:
`niceness = 5` seems redundant to me, because of `--nice 5`
```suggestion
- Start a daemon with lowered priority that can run a maximum of 4 tasks at a time:
```